app/graphql/context.tsx

```diff
@@ -4,6 +4,17 @@ import { GRAPHQL_ENDPOINT } from "../domain/env";
 
 const client = createClient({
   url: GRAPHQL_ENDPOINT,
+  fetchOptions: () => {
+    const lang =
+      typeof document !== "undefined"
+        ? document.querySelector("html")?.getAttribute("lang")
+        : undefined;
+    return {
+      headers: {
+        "Accept-Language": lang || "de",
```

Comment:
Nit: Replace "de" with the `defaultLocale`  from locales/locales.ts

Reply:
Good point, done in step 2 👍